build.gradle

```diff
@@ -51,13 +51,7 @@ ext.targetCompatibility = JavaVersion.VERSION_11
 
 tasks.withType(JavaCompile) {
     options.compilerArgs += ["--add-modules", "java.se",
-                             "--add-exports", "java.base/jdk.internal.ref=ALL-UNNAMED",

```

Comment:
This looks like just compiler args so might be fine, but it is my understanding that the se are required for optimal operation: https://docs.hazelcast.org/docs/latest/manual/html-single/#running-in-modular-java

Reply:
Oh, looks like they're already in the jvmArgs segment